### PR TITLE
adds incidents/2020-03-24--iiif-prod-outage.md

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: incidents/2020-03-24--iiif-prod-outage.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# iiif outage
+
+* affected iiif--prod--* 
+* affected for approximately 4 hours.
```
Comment (giorgiosironi) [inline incidents/2020-03-24--iiif-prod-outage.md:6]: can be separated into `Impact`, see https://github.com/elifesciences/tech-team/blob/master/incidents/2018-04-24-journal-outage.md#impact

File: incidents/2020-03-24--iiif-prod-outage.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+43.249.75.20 - - [23/Mar/2020:21:16:00 +0000] "HEAD /ping-fastly HTTP/1.1" 200 0 "-" "Varnish/fastly (healthcheck)" 0.000
+157.52.104.43 - - [23/Mar/2020:21:16:00 +0000] "HEAD /ping-fastly HTTP/1.1" 200 0 "-" "Varnish/fastly (healthcheck)" 0.000
+157.52.79.35 - - [23/Mar/2020:21:16:00 +0000] "GET /lax:08419%2Felife-08419-fig4-figsupp5-v2.tif/full/full/0/default.jpg HTTP/1.1" 200 209689 "-" "GuzzleHttp/6.2.1 curl/7.38.0 PHP/7.1.28" 0.187
+157.52.101.23 - - [23/Mar/2020:21:16:01 +0000] "HEAD /ping-fastly HTTP/1.1" 200 0 "-" "Varnish/fastly (healthcheck)" 0.000
+10.0.2.46 - - [24/Mar/2020:01:17:52 +0000] "GET / HTTP/1.1" 200 220 "-" "ELB-HealthChecker/1.0" 0.004
+10.0.3.69 - - [24/Mar/2020:01:17:53 +0000] "GET / HTTP/1.1" 200 220 "-" "ELB-HealthChecker/1.0" 0.002
+127.0.0.1 - - [24/Mar/2020:01:17:57 +0000] "GET / HTTP/1.1" 200 220 "-" "curl/7.58.0" 0.012
+```
+
+
```
Comment (giorgiosironi) [inline incidents/2020-03-24--iiif-prod-outage.md:53]: Any [corrective action](https://github.com/elifesciences/tech-team/blob/master/incidents/2018-04-24-journal-outage.md#corrective-actions) to add?

Comment (lsh-0) [reply incidents/2020-03-24--iiif-prod-outage.md:53]: not unless you can think of something

Comment (giorgiosironi) [reply incidents/2020-03-24--iiif-prod-outage.md:53]: Just link https://github.com/elifesciences/issues/issues?q=is%3Aopen+is%3Aissue+label%3Aiiif if you'd like.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**